docs/pages/tutorial/emojimon/deploy-to-testnet.mdx

```diff
@@ -0,0 +1 @@
+# Deploy to testnet
```

Comment:
testnet instructions haven't changed much, wondering if we can just copy that over from the old tutorial